server/data/prisonerSearchClient.test.ts

```diff
@@ -79,21 +79,16 @@ describe('prisonSearchClientBuilder', () => {
 
   describe('getPrisonerById', () => {
     it('should return data for single prisoner by prisoner ID', async () => {
-      const results = {
-        lastName: 'FORENAME',
-        firstName: 'SURNAME',
-        prisonerNumber: 'A1234BC',
-        dateOfBirth: '2000-01-01',
-        prisonId: 'HEI',
-        prisonName: 'HMP Hewell',
-        cellLocation: '1-1-C-028',
-        restrictedPatient: false,
-      } as Prisoner
-      fakePrisonerSearchApi.get('/prisoner/A1234BC').matchHeader('authorization', `Bearer ${token}`).reply(200, results)
+      const prisoner = createPrisoner()
+
+      fakePrisonerSearchApi
+        .get('/prisoner/A1234BC')
```

Comment:
My only suggestion here would be there is no link between the prisonerId in the get and createPrisoner, maybe if you passed an id to the latter it creates more of an understandable link? If that makes sense?

Reply:
Yeah, good suggestion; that would make it clearer. Or, alternatively use something like `.get('/prisoner/${prisoner.offenderNo}')`

Trying to chip away at rationalising and tidying the tests so will add to the list!

Reply:
Aye tis awesome work 👍